lib/boot-without-auth.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import { Auth as AuthApp } from './auth';
+import { Auth as SimperiumAuth } from 'simperium';
+import analytics from './analytics';
+
+import getConfig from '../get-config';
+
+import '../scss/style.scss';
+
+type Props = {
+  onAuth: (token: string, username: string, createWelcomeNote: boolean) => any;
+};
+
+type State = {
+  authStatus:
+    | 'unsubmitted'
+    | 'submitting'
+    | 'invalid-credentials'
+    | 'unknown-error';
+};
+
+type User = {
+  access_token?: string;
+};
+
+const appProvider = 'simplenote.com';
+const config = getConfig();
+const auth = new SimperiumAuth(config.app_id, config.app_key);
+
+class AppWithoutAuth extends Component<Props, State> {
+  state: State = {
+    authStatus: 'unsubmitted',
+  };
+
+  authenticate = (username: string, password: string) => {
+    if (!(username && password)) {
+      return;
+    }
+
+    this.setState({ authStatus: 'submitting' }, () => {
+      auth
+        .authorize(username, password)
+        .then((user: User) => {
+          if (!user.access_token) {
+            throw new Error('missing access token');
+          }
+          this.props.onAuth(user.access_token, username, false);
+        })
+        .catch(({ message }: Error) => {
+          if (
+            'invalid password' === message ||
+            message.startsWith('unknown username:')
+          ) {
+            this.setState({ authStatus: 'invalid-credentials' });
+          } else {
+            this.setState({ authStatus: 'unknown-error' });
+          }
+        });
+    });
+  };
+
+  createUser = (username: string, password: string) => {
+    if (!(username && password)) {
+      return;
+    }
+
+    this.setState({ authStatus: 'submitting' }, () => {
+      auth
+        .create(username, password, appProvider)
+        .then((user: User) => {
+          if (!user.access_token) {
+            throw new Error('missing access token');
+          }
+
+          analytics.tracks.recordEvent('user_account_created');
+          this.props.onAuth(user.access_token, username, true);
+        })
+        .catch(() => {
+          this.setState({ authStatus: 'unknown-error' });
+        });
+    });
+  };
+
+  tokenLogin = (username: string, token: string) => {
+    this.props.onAuth(token, username, false);
+  };
+
+  render() {
+    const systemTheme = window.matchMedia('(prefers-color-scheme: dark)')
```

Comment:
I think this needs to be smarter pulling from localstorage before checking system theme.

Reply:
I'm happy to leave this for another PR, I'm not sure how it currently works.

Reply:
Note that at this point we aren't logged in. In the existing code we don't clear settings and so that stays in `localStorage`, which while convenient for places like this also seems wrong. If we really want to log out then I think we should forget the settings that person may have saved.

Do you feel differently?

Reply:
If you delete the settings that person set then every time they log in they will have to go change their settings. Until we have a way to preserve settings then I think it cannot be cleared.